src/test/kotlin/ApplicationLocalPostgres.kt

```diff
@@ -18,16 +18,17 @@ class ApplicationLocalPostgres
 fun main(args: Array<String>) {
 
     val sqlContainer = KPostgreSQLContainer("postgres")
-            .withDatabaseName("familie-oppdrag")
+            .withDatabaseName("familie-ef-sak")
             .withUsername("postgres")
             .withPassword("test")
 
     sqlContainer.start()
 
     val properties = Properties()
-    properties["spring.datasource.url"] = sqlContainer.jdbcUrl
+    properties["SPRING_DATASOURCE_URL_OVERRIDE"] = sqlContainer.jdbcUrl
     properties["spring.datasource.username"] = sqlContainer.username
     properties["spring.datasource.password"] = sqlContainer.password
+    properties["spring.datasource.driver-class-name"] = "org.postgresql.Driver"
```

Comment:
Burde vi, for "sikkerhets skyld" sette samme property på disse for å være sikker på at de ikke blir overskrevet senere?